update.go

```diff
@@ -151,8 +156,9 @@ other options are ignored.
 			for opt, dest := range map[string]*uint64{
 				"cpu-period":         r.CPU.Period,
 				"cpu-quota":          r.CPU.Quota,
-				"cpu-share":          r.CPU.Shares,
+				"cpu-shares":         r.CPU.Shares,
```

Comment:
> The main intent of PR is to add the kernelTCP limits

so that flag change would need its own commit, I'll let other maintainers chime in though..